### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by leanazulyoro**
~~(Leaving as draft until #1 gets merged)~~

This PR adds a deployment workflow to deploy to Hostinger.
It adds config needed to run under a subdirectory (/uniswap)

Other changes:
- ~~thDAI renamed to hDAI~~
- Disabled the setting to change theme (light/dark), until we have a complete dark mode.

### CODE REVIEW

**Review by dhidalgX — DISMISSED** [review]
deployment.yaml is correct and uniswap directory haven been created in Hostinger

**Comment by gndelia** [comment]
> thDAI renamed to hDAI

`thDAI` should be the correct name on the UI for Hemi Sepolia. I just checked and the contract returns `hDAI`, which I think is incorrect (it doesn't follow the [token naming conventions defined in our source of truth](https://github.com/hemilabs/infrastructure/blob/main/NETWORK_INFO.md#token-naming-standards). We should either use the proper value, or update our naming convention.  @jcvernaleo  Thoughts?

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Deploy Hemi Uniswap
+
+on:
+  workflow_dispatch:
```
Comment (gndelia) [inline .github/workflows/deployment.yml:4]: Is the idea to deploy manually?

Comment (leanazulyoro) [reply .github/workflows/deployment.yml:4]: I actually copied this from pure.finance, I didn't even realize it 😅. According to @gabmontes, it doesn't hurt to leave for now.

**Comment by leanazulyoro** [comment]
@gndelia I removed the commit that renamed thDAI to hDAI for now, we can add that in another PR if we decide to go along with it.

**Review by gndelia — APPROVED** [review]

**Pull request merged**